ptbcontrib/error_log/log_handler.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+#
+# A library containing community-based extension for the python-telegram-bot library
+# Copyright (C) 2020-2023
+# The ptbcontrib developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""This module contains a helper function to pipe logging messages to Telegram chat."""
+from logging import BASIC_FORMAT, Formatter, LogRecord, StreamHandler
+from typing import List
+
+from telegram import Bot
+
+
+class ErrorBroadcastHandler(StreamHandler):
+    """
+    Allows to send records from logging module to a specified Telegram chat.
+
+    Args:
+        bot (:obj:`Bot`): The bot object
+        levels (:obj:`List[int]`): List of logging levels caught by this handler
+    """
+
+    def __init__(
+        self,
+        bot: Bot,
+        levels: List[int],
+        chat_id: int,
+        log_format: str = BASIC_FORMAT,
+        # pylint: disable=R0913, W0102
+        send_message_kwargs: dict = {},
```

Comment:
thinking if `chat_id` should go into `send_message_kwargs` (and make them actual kwargs)